gensrc/thrift/Descriptors.thrift

```diff
@@ -344,12 +344,33 @@ struct TFileTable {
     2: optional list<TColumn> columns
 }
 
+struct TIcebergSchema {
+    1: optional list<TIcebergSchemaField> fields
+}
+
+struct TIcebergSchemaField {
+    // Refer to field id in iceberg schema
+    1: optional i32 field_id
```

Comment:
Seems it's better to add this in `TColumn`. So we don't have to define it twice.

Reply:
`TColumn` is not only used for IcebergTable, I think it's better to put it in IcebergTable.